lib/cli/commands/build.js

```diff
@@ -52,28 +52,38 @@ optimized output compatible with Node 6.
     });
   }
 
-  build(dirs = this.dirs) {
-    if (!dirs) {
-      dirs = this.dirs = [ 'app', 'config' ];
+  build(globs = this.globs) {
+    if (!globs) {
+      globs = this.globs = [
+        'app/**/*',
+        'config/**/*',
+        'blueprints/*/index.js',
+        { glob: 'blueprints/*/files/**/*', transform: false }
```

Comment:
Ah, I missed the fact that this block will copy the file over when it doesn't end it .js.

Yea, this will work for now, but I'd definitely like to explore converting to a broccoli base build process.